qiskit/providers/aer/extensions/multi_pauli.py

```diff
@@ -0,0 +1,75 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2020
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""
+Simulator command to perform multiple pauli gates in a single pass
+"""
+import numpy as np
+
+from qiskit import QuantumCircuit, QuantumRegister
+from qiskit.circuit.library import XGate, YGate, ZGate
+from qiskit.circuit.gate import Gate
+
+
+class MultiPauliGate(Gate):
```

Comment:
The `Pauli` op is already a multi-qubit Pauli, so I think this should just be called `PauliGate` not `MultiPauliGate`

Reply:
Agreed.